common/men_at_arms_types/00_maa_types.txt

```diff
@@ -126,6 +130,35 @@ pikemen_unit = {
 	icon = pikemen
 }
 
+caster_unit = {
```

Comment:
Move new unit types at the end of the file or/and mark them by the comment `#Warcraft` so they don't mix with vanilla ones.

Reply:
Nvm, I get now. They're sorted by era. Anyway, mark them by the comment.